### ISSUE TITLE
Socialize Discovery AX+ HCD sheet with Design team

### ISSUE THREAD

**Issue by niradatta1** [CivicActions/accessibility#482]
Need to identify and draft templates that will eventually be used as CA branded templates we can leverage for client work on accessibility. Focusing on Discovery phase. Ideally, these should be established for each phase identified in the HCD doc: 

https://docs.google.com/document/d/1rFb1sYwSYBicRlz1CU_hEWQRV8v-t_UJ-rp1ZKfqPSI/edit#

Next step: Socialize with internal team and iterate on the templates.
